enforcer/utils/rpcwrapper/types.go

```diff
@@ -0,0 +1,104 @@
+package rpcwrapper
+
+import (
+	"time"
+
+	"github.com/aporeto-inc/trireme/enforcer"
+	"github.com/aporeto-inc/trireme/enforcer/utils/tokens"
+	"github.com/aporeto-inc/trireme/policy"
+)
+
+var gobTypes = []interface{}{
+	InitRequestPayload{},
+	InitResponsePayload{},
+	InitSupervisorPayload{},
+	EnforcePayload{},
+	UnEnforcePayload{},
+	SuperviseRequestPayload{},
+	UnSupervisePayload{},
+}
+
+//Request exported
+type Request struct {
+	MethodIdentifier int
+	HashAuth         []byte
+	Payload          interface{}
+}
+
+//exported
+const (
+	SUCCESS      = 0
+	StatsChannel = "/tmp/statschannel.sock"
+)
+
+//Response exported
+type Response struct {
+	MethodIdentifier int
+	Status           error
```

Comment:
Status should not be an error, create a new type where we could have "FAILED" and "SUCCESS"